discovery-provider/alembic/versions/aab240348d73_add_premium_content_columns.py

```diff
@@ -0,0 +1,50 @@
+"""add-premium-content-columns
+
+Revision ID: aab240348d73
+Revises: 5d3f95470222
+Create Date: 2022-08-22 20:22:22.439424
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "aab240348d73"
+down_revision = "551f5fc03862"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "tracks",
+        sa.Column("is_premium", sa.Boolean(), nullable=False, server_default="false"),
+    )
+    op.add_column(
+        "tracks",
+        sa.Column("premium_conditions", postgresql.JSONB(), nullable=True),
+    )
+
+    connection = op.get_bind()
+    connection.execute(
+        """
+        begin;
+            CREATE INDEX IF NOT EXISTS track_is_premium_idx ON tracks (is_premium);
```

Comment:
i actually don't think this would be necessary for Entity Manager because we would fetch all relevant tracks at first and then pass it in to the access check logic like i mentioned below

Reply:
maybe if we need to query on this for the API or the new streamable one https://github.com/AudiusProject/audius-protocol/pull/3729#event-7222992920

Reply:
makes sense

Reply:
probably (is_premium, is_current, is_delete)